### DISCUSSION TITLE
"the connection is not allowed. to further scamming, we only allow connections between users in the same city."

### DISCUSSION THREAD

**Discussion by DavidMR91** [rustdesk/rustdesk#7938] [General]
I have tried to connect to various remote desktops and I get the following message: "the connection is not allowed. to further scamming, we only allow connections between users in the same city."
What is happening with Rustdesk?

**Comment by quarter08** [comment]
Same problem here. I hope that is not the future of rustdesk :)

**Reply by quarter08** [reply]
https://twitter.com/rustdesk/status/1787689958276472982?t=Cn1RQ7S6OpZsa8UfqohbNA&s=19
